lib/models/class.dart

```diff
@@ -0,0 +1,2 @@
+///Class extends event class
+/// @author Praneet Singh
```

Comment:
Looks good! Thanks for following the doc comment format that I used in the Event! Go ahead with the rest of the class